app/src/components/ViewEvent/ViewEventContainer.tsx

```diff
@@ -181,9 +181,13 @@ export const ViewEventContainer = ({ eventId }: IProps) => {
     );
   };
 
+  const addChristmasSpirit = ['🎅', '🧑‍🎄', '🤶', 'christmas'].some((emoji) =>
+    event.title.toLowerCase().includes(emoji)
+  );
+
   return (
     <>
-      {remoteEvent.data.shortname === 'bekk-christmas-2021' && <Snow />}
```

Comment:
Kom på at vi må gjøre noe ekstra hvis vi vil ha fargene på toppen riktig også 😊 Det er en annen sjekk i koden som sjekker på hardkoda eventId for bekk-christmas-2021 eventet, for å gi det rød topp. Der har vi ikke tilgang til hele event-objektet, så må hentes først.